src/components/settings-panel.tsx

```
@@ -154,9 +161,55 @@ const SettingsPanelContent = ({
                 </SelectContent>
               </Select>
             </div>
+            {/* Reasoning Effort */}
+            <div className="flex items-center justify-between gap-2">
+              <Label htmlFor={`${id}-reasoning`} className="font-normal flex items-center gap-1">
+                Reasoning Effort
+                <span className="relative group ml-1">
+                  <span
+                    tabIndex={0}
```

Biome (1.9.4)

[error] The HTML element span is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)